src/components/Calculator.jsx

```diff
@@ -0,0 +1,63 @@
+const Buttons = () => (
+  <div className="wrapper">
+    <div className="screen">0</div>
+    <div className="btn light-gray">
+      AC
+    </div>
+    <div className="btn light-gray">
+      +/-
+    </div>
+    <div className="btn light-gray">
+      %
+    </div>
+    <div className="btn orange">
+      /
+    </div>
+    <div className="btn">
```

Comment:
- [ ] According to the project requirements, the Calculator.js component should have at least one child component. Kindly create a separate component that you will use inside this component as a child component. 👍🏽

<img width="1042" alt="Screenshot 2023-06-06 at 11 38 25 AM" src="https://github.com/rahimialijan/math-magicians/assets/50941074/76573238-832c-4da8-9f62-024680c7e2b2">